.env.example

```
@@ -16,3 +16,12 @@ HANTU_APPKEY=your-hantu-appkey
 HANTU_APPSECRET=your-hantu-appsecret
 
 OPENAI_API_KEY=your-openai-api-key
+
+BOK_API_KEY=your-bok-api-key
+
+TOSS_SECRET_KEY=your-toss-secret-key
+
+MAIL_USERNAME=your-mail-username
+MAIL_PASSWORD=your-mail-pw
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MAIL_PASSWORD key should go before the MAIL_USERNAME key

(UnorderedKey)